readsettings/__init__.py

```diff
@@ -19,51 +19,43 @@ def _mkdir(path):
 class ReadSettings:
     """
     Main ReadSettings class.
-
     :type path: string
     :param path: The settings file to use.
-
     :type ext: string
     :param ext: Override the file type.
-
     :type autosave: boolean
     :param autosave: Set the autosave behaviour. Default is True.
-
     :raises ValueError: Invalid file type provided!
-
     >>> data = ReadSettings(".rs-tmp/t1.json")
     >>> data["foo"] = "Hello World"
     >>> data["foo"]
     'Hello World'
     >>> del data["foo"]
-
     >>> data = ReadSettings(".rs-tmp/t0.invalid")
     Traceback (most recent call last):
       ...
     ValueError: Invalid file type provided!
-
     >>> data = ReadSettings(".rs-tmp/t6.json")
     >>> data["helloWorld"] = "newValue"
-
     >>> data = ReadSettings(".rs-tmp/t1.yml")
     >>> data["helloWorld"] = "newValue"
-
     >>> data = ReadSettings(".rs-tmp/t1.toml")
     >>> data["helloWorld"] = "newValue"
     """
 
-    def __init__(self, path, ext=None, autosave=True):
+    def __init__(self, path, ext=None, autosave=True, encoding=None):
```

Comment:
What if you did: `encoding=sys.getdefaultencoding()`?

Reply:
> Is it better to force `encoding='utf-8'`?
Considering that some Windows users have already used this to write configuration files, this keeps upward compatibility.